install.sh

```diff
@@ -1,9 +1,103 @@
+commands="curl kubectl"
 
-curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
-chmod 700 get_helm.sh
-./get_helm.sh
+
+trap cleanup EXIT
+echo "[PRE-INSTALL]: Checking all pre-requisites are met"
+# Iterate the string variable using for loop
+for val in $commands; do
+    if command -v $val > /dev/null; then
+      echo "[PRE-INSTALL]: $val correctly installed"
+    else
+      echo "[PRE-INSTALL] [FATAL] : $val not found. Exiting"
+      exit 1
+    fi
+done
+
+set -e
+
+mkdir -p /tmp/liqo
+cd /tmp/liqo || exit 1
+mkdir -p /tmp/liqo
+cd /tmp/liqo || exit 1
+mkdir -p bin/
+cp linux-amd64/helm bin/
+export OLD_PATH=$PATH
+export PATH=$PATH:/tmp/liqo/bin/
+echo " [PRE-INSTALL] [HELM] Checking HELM installation..."
+HELM_INSTALLATION=false
+  if command -v helm > /dev/null; then
+    HELM_INSTALLED_VERSION=$(helm version --template "{{ .Version }}")
+    echo "[PRE-INSTALL]: Helm, found version: $HELM_INSTALLED_VERSION"
+    if [ "$HELM_INSTALLED_VERSION" != "$HELM_VERSION" ]; then
+         HELM_INSTALLATION=true
+         echo " [PRE-INSTALL] [HELM] found but having a potential incompatible version... Installing it"
+    fi
+  else
+    echo " [PRE-INSTALL] [HELM]: Helm not found... Installing it"
+    HELM_INSTALLATION=true
+fi
+if [ $HELM_INSTALLATION = true ]; then
+    echo " [PRE-INSTALL] [HELM]: Installing Helm $HELM_VERSION"
+    curl --fail -LO $HELM_URL > /dev/null
+    tar zxf helm-v3.2.3-linux-amd64.tar.gz
+fi
+
+if [ ! -d "$FOLDER" ] ; then
+    git clone "$URL" 
+fi
+
+echo "[PRE-INSTALL]: Collecting installation variables. The installer will retrieve installation parameters from your
+ Kubernetes cluster. Fill free to override them, by launching it with those environment variables set in advance."
+if [ -z "$POD_CIDR" ]
+then
+  echo "[PRE-INSTALL]: Kubeconfig variable is set to: $KUBECONFIG"
+else
+  echo "[PRE-INSTALL]: Kubeconfig variable is set to: ~/.kube/config"
+fi
+if [ -z "$POD_CIDR" ]
+then
+    POD_CIDR=$(kubectl cluster-info dump | grep -m 1 cluster-cidr | sed 's/--cluster-cidr=//' | sed 's/[,"]//g' | sed 's/ //g')
+fi
+echo "[PRE-INSTALL]: POD_CIDR is set to: $POD_CIDR"
+
+if [ -z "$SERVICE_CIDR" ]
+then
+    SERVICE_CIDR=$(kubectl cluster-info dump | grep -m 1 service-cluster-ip-range | sed 's/--service-cluster-ip-range=//' | sed 's/[,"]//g' | sed 's/ //g')
+fi
+echo "[PRE-INSTALL]: SERVICE_CIDR is set to: $SERVICE_CIDR"
+
+if [ -z "$GATEWAY_PRIVATE_IP" ]
+then
+    GATEWAY_PRIVATE_IP=192.168.1.1
+fi
+echo "[PRE-INSTALL]: GATEWAY_PRIVATE_IP is set to: $GATEWAY_PRIVATE_IP"
+
+if [ -z "$GATEWAY_IP" ]
+then
+    GATEWAY_IP=$(kubectl get no -l "liqonet.liqo.io/gateway=true" -o jsonpath='{.items[0].status.addresses[0].address}')
+fi
+echo "[PRE-INSTALL]: GATEWAY_IP is set to: $GATEWAY_IP"
+
+echo "COMMAND: helm install liqo -n liqo ./ --set podCIDR=$POD_CIDR --set serviceCIDR=$SERVICE_CIDR --set gatewayPrivateIP=$GATEWAY_PRIVATE_IP --set gatewayIP=$GATEWAY_IP"
```

Comment:
This should be a variable, otherwise soon it will be different from the actually executed command